util/venv-dependencies.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -e
+
+# quiet=--quiet
```

Comment:
It might be worth documenting the `quiet` behaviour with a comment like was done in the other script.